[DOCS] Grouped nav menu updates (BIG edition) (#2279)
elastic/security-docs: docs/getting-started/ingest-data.asciidoc

* The {fleet-guide}/fleet-overview.html[{agent}] with the **Elastic {endpoint-cloud-sec}** integration, which protects
your hosts and sends logs, metrics, and endpoint security data to {elastic-sec}. See <<install-endpoint>>.

> * The {agent} with integrations, which are available in the {fleet-guide}/fleet-overview.html#package-registry-intro[Elastic Package Registry (EPR)]. To install an integration that works with {elastic-sec}, select **Security** from the main nav menu and click *Add security integrations*. Alternatively, go to the {kib} Home page and click *Add integrations*. On the Integrations page, click the *Security* category filter, then select an integration to view the installation instructions. For more information on integrations, refer to {integrations-docs}[{integrations}].

Reviewer: It looks like the Kibana menu has changed a bit too, so some of this can be combined:
```suggestion
* The {agent} with integrations, which are available in the {fleet-guide}/fleet-overview.html#package-registry-intro[Elastic Package Registry (EPR)]. To install an integration that works with {elastic-sec}, go to the {kib} Home page or main navigation menu and click *Add integrations*. On the Integrations page, click the *Security* category filter, then select an integration to view the installation instructions. For more information on integrations, refer to {integrations-docs}[{integrations}].
```
